tests/test_dashboard_page.py

```diff
@@ -2,7 +2,9 @@
 
 from pages.BasePage import BasePage
 from pages.DashboardPage import DashboardPage
-from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators
+from config.TestData import TestData as TD
+from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators, \
+    Titles
 
 
 class TestHomePage:
```

Comment:
There is no such page as "HomePage"
i think it should be change to "DashboardPage"

Reply:
ok